interface/forms/soap/templates/general_new.html

```diff
@@ -2,13 +2,43 @@
 <head>
 <title>{xlt t='SOAP'}</title>
 {headerTemplate}
+    {literal}
+    <script>
+        // handles the call back from the popup
+        function set_related(codetype, code, selector, codedesc) {
+            if (code) {
+                if (codetype == 'ICD10') {
+                    document.getElementById('dxcode').value += code + " - " + codedesc + " ,, \r\n";
+                }
+                if (codetype == 'CPT4') {
+                    document.getElementById('cpt').value += code + " - " + codedesc + " ,, \r\n";
+                }
+                let xhr = new XMLHttpRequest();
+                xhr.onload = function() {
```

Comment:
This seems odd, was this just for debugging?  Or were you still working on this?

Reply:
If you are referring to the console log, not sure why that is there either other than I wanted to see a return during the build process. I am removing it. It's not needed.